plugin/trino-elasticsearch/src/test/java/io/trino/plugin/elasticsearch/ElasticsearchQueryRunner.java

```diff
@@ -128,8 +127,8 @@ public static Session createSession()
     public static void main(String[] args)
             throws Exception
     {
-        Logging.initialize();
-
+        // To start Elasticsearch:
+        // docker run -p 9200:9200 -e "discovery.type=single-node" docker.elastic.co/elasticsearch/elasticsearch:7.6.2
```

Comment:
We can simply remove this line instead of a separate commit.

@nineinchnick By the way, I added `ElasticsearchExternalQueryRunner` few days ago. Could you remove the logging initialization from the class if necessary?

Reply:
This is a bad rebase, sorry, I should pay more attention. Thanks for spotting this. I applied the changes in the other runner.